packages/binding-mbus/src/mbus-connection.ts

```diff
@@ -182,21 +182,25 @@ export class MBusConnection {
                 // inform all the operations that the connection cannot be recovered
                 this.queue.forEach((transaction) => {
                     transaction.operations.forEach((op) => {
-                        op.failed(error);
+                        op.failed(`${error}`);
```

Comment:
In Modbus we are using `op.failed(err instanceof Error ? err : new Error(JSON.stringify(err)))` I think we should be consistent.

Reply:
Maybe we could create a helper function to be consistent across packages here?

Reply:
done